pkg/services/sqlstore/user.go

```diff
@@ -150,7 +152,7 @@ func CreateUser(cmd *m.CreateUserCommand) error {
 		}
 
 		return nil
-	})
+	}, 0)
```

Comment:
This change is looking much cleaner now 🙂 I do still wonder whether we should provide `inTransactionCtx` that can then call `inTransactionWithRetryCtx(..., 0)` (in the same way `inTransaction` calls `inTransactionWithRetry`) to avoid having to add the `, 0` all over the place.

Reply:
Yeap. I added that function in the `alert_reminder` branch to make things cleaner. might as well add it here since they might not be merged at the same time.